skins/oasis/modules/templates/TemplateDraftModule_Approve.php

```diff
@@ -1,9 +1,13 @@
 <section class="templatedraft-module module">
 	<h2><?= wfMessage( 'templatedraft-module-title-approve' )->escaped() ?></h2>
 	<p><?= wfMessage( 'templatedraft-module-content-approve' )->escaped() ?></p>
-	<a href="<?= $draftUrl ?>">
-		<button class="templatedraft-module-button" type="button" data-id="templatedraft-module-button-approve">
-			<?= wfMessage( 'templatedraft-module-button-approve' )->escaped() ?>
-		</button>
-	</a>
+	<?php if ( $allowApprove ): ?>
+		<a href="<?= $draftUrl ?>">
```

Comment:
As a best practice, `Sanitizer::encodeAttribute` should be used here, even though it's URL encoded. Additionally HTML escaping provides correctness to the output. And though it's very unlikely in this case, what if someone changed how `$draftUrl` was constructed and suddenly a parameter or another part of the URL that a user controls isn't URL encoded? Better to always escape unless you need it to not be, which is why templating languages like Mustache with `{{}}` provide advantages due to most people defaulting to the escaped version without extra work.